modules/release-notes/pages/0.7.0-rn.adoc

```diff
@@ -101,11 +102,11 @@ Reworks the asset installer in `+dfx+` to use these methods. It can therefore up
 `+--wallet+` allows users to specify a particular wallet to use for calls. 
 * Implement the HTTP request proposal in `+dfx+` bootstrap webserver
 +
-Ads support for http requests in the base storage canister (with a default to `+/index.html+`).
+Adds support for http requests in the base storage canister (with a default to `+/index.html+`).
 
 This does not support encodings other than `+identity+`, and doesn't return any headers. 
 
-This also upgrades tokio and request in order to work correctly. There are some performance issues to note (this is slower than the icx-http-server).
+This also upgrades `+tokio+` and `+reqwest+` in order to work correctly. There are some performance issues to note (this is slower than the icx-http-server).
```

Comment:
The correct spelling of the library is ``. I take the blame for letting it become (in #503) "request"  in a recent commit. This fixes it back to good.

Reply:
Thank you!